website/docs/r/app_service_virtual_network_swift_connection.html.markdown

```diff
@@ -9,11 +9,27 @@ description: |-
 
 # azurerm_app_service_virtual_network_swift_connection
 
-Manages an App Service Virtual Network Association (this is for the [Regional VNet Integration](https://docs.microsoft.com/azure/app-service/web-sites-integrate-with-vnet#regional-vnet-integration)).
+Manages an App Service Virtual Network Association for [Regional VNet Integration](https://docs.microsoft.com/azure/app-service/web-sites-integrate-with-vnet#regional-vnet-integration).
 
-~> **Note:** This resource can be used for both `azurerm_app_service` and `azurerm_function_app`.
+This resource can be used for both `azurerm_app_service` and `azurerm_function_app`.
```

Comment:
since those resources are both deprecated perhaps we should make this:

```suggestion
This resource can be used for both App Services and Function Apps.
```